spec/features/guest/guest_show_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+RSpec.describe "the guest show page", type: :feature do
+
+  it "shows a list of all the rooms a guest has stayed in" do
+    hotel_1 = Hotel.create!(name: 'Echo Mountain Inn', location: 'Echo Mountain')
+    room_1 = hotel_1.rooms.create!(rate: 125, suite: "Presidential")
+    room_2 = hotel_1.rooms.create!(rate: 5, suite: "Squatter")
+    guest_1 = Guest.create!(name: 'Charlize Theron', nights: 3)
+
+    guest_1.rooms << room_1
+    guest_1.rooms << room_2
+    # When I visit a guest's show page
+    visit "/guests/#{guest_1.id}"
+    # save_and_open_page
+    # I see the guest's name
+    expect(page).to have_content(guest_1.name)
+    # And I see a list of all the rooms they've stayed in including the room's suite, nightly rate, 
+    expect(page).to have_content(room_1.suite)
+    expect(page).to have_content(room_2.suite)
+    expect(page).to have_content(room_1.rate)
```

Comment:
These assertions are good, but could be more robust if you included the text label they're anchored to, i.e. expecting the page to have "Room Rate: 125" instead of just "125" - this will lead to fewer possibilities of false positives with these tests